spring-integration-file/src/main/resources/org/springframework/integration/file/config/spring-integration-file-4.2.xsd

```diff
@@ -101,6 +101,17 @@ Only files matching this regular expression will be picked up by this adapter.
                     </xsd:appinfo>
                 </xsd:annotation>
             </xsd:attribute>
+			<xsd:attribute name="ignore-hidden">
+				<xsd:annotation><xsd:documentation><![CDATA[
+					A boolean flag indicating whether hidden files shall be ignored.
+					If set to 'false', hidden files will be processed. If not specified,
+					this value will default to 'true' and an 'IgnoreHiddenFileListFilter'
+					will be added (Since Spring Integration 4.2).
```

Comment:
No need to to say `since` in the XSD.
At least we haven't done similar before, so I don't see reason to start doing this at all.
The `What's New` and a target Reference chapter are good places to demonstrate our writing talent :smile: